wilkes/roster.json

```diff
@@ -124,7 +124,12 @@
     },
     {
       "name": "Arvin Jhurry",
-      "github": "ArvinJhurry"
+      "github": "ArvinJhurry",
+      "materials": [
+        {
+          "type":"cv"
+      }
+      ]
```

Comment:
Can you please make sure your code is properly indented :)?

Reply:
Sorry about that